### PR TITLE
[DE-436] Extract segments models

### PR DESCRIPTION
**Pull request by maciej-nedza**

### CODE REVIEW

File: reference/Chargify-API.v1.yaml [hunk 1]
```diff
@@ -18344,76 +18344,7 @@ paths:
...
-                            unit_price:
-                              type: string
-                            price_point_id:
-                              type: integer
-                            formatted_unit_price:
-                              type: string
-                            segment_id:
-                              type: integer
-              examples: {}
+                $ref: "../components/schemas/Segment.yaml"
```
Comment (michalpierog) [inline reference/Chargify-API.v1.yaml:18347]: Previously it was wrapped in segment, is it correct?

File: reference/Chargify-API.v1.yaml [hunk 2]
```diff
@@ -18704,75 +18511,7 @@ paths:
...
-                                - "null"
-                            unit_price:
-                              type: string
-                            price_point_id:
-                              type: integer
-                            formatted_unit_price:
-                              type: string
-                            segment_id:
-                              type: integer
+                $ref: "../components/schemas/Segment.yaml"
```
Comment (michalpierog) [inline reference/Chargify-API.v1.yaml:18514]: Same here, unwrapped model used.

**Review by michalpierog — APPROVED** [review]
LGTM, added a few comments

File: reference/Chargify-API.v1.yaml (hunk 1 above)
Comment (maciej-nedza) [reply reference/Chargify-API.v1.yaml]: Indeed, missed that

**Pull request merged**